frontend/package.json

```diff
@@ -29,6 +31,7 @@
     "@types/react-router-dom": "^5.3.3",
     "@types/superagent": "^4.1.21",
     "@types/uuid": "^9.0.7",
+    "@types/mocha": "^10.0.6",
```

Comment:
Why are `mocha` types needed? 🤔  It is used by cypress, maybe?

Reply:
Because it helps my IDE to understand the typing of the component test files. Probably not the best way, if you have a better suggestion it is welcome :)